README.md

```diff
@@ -8,6 +18,26 @@ To get started see the [Azure Bot Service Documentation](https://docs.microsoft.
 
 A rich set of samples are available at [BotBuilder-Samples](https://github.com/microsoft/botbuilder-samples) repository.
 
+
+## Packages
+| Name                                  | Released Package | Daily Build                                                                                                                                                                  |
+|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|--|
+| botbuilder                 | [![BotBuilder Badge](https://img.shields.io/npm/dt/botbuilder.svg?logo=npm&label=npm)](https://www.npmjs.com/package/botbuilder/)                                 | TBD |
+| botbuilder-ai        | [![BotBuilder Badge](https://img.shields.io/npm/dt/botbuilder-ai.svg?logo=npm&label=npm)](https://www.npmjs.com/package/botbuilder-ai/)                 | TBD |
+| botbuilder-azure          | [![BotBuilder Badge](https://img.shields.io/npm/dt/botbuilder-azure.svg?logo=npm&label=npm)](https://www.npmjs.com/package/botbuilder-azure/)                   | TBD |
```

Comment:
instead of the "label=npm" on each of these, please use the package name.  That means using this:
`https://img.shields.io/npm/dt/botbuilder-azure.svg?logo=npm&label=botbuilder-azure`

[![BotBuilder Badge](https://img.shields.io/npm/dt/botbuilder-azure.svg?logo=npm&label=botbuilder-azure)](https://www.npmjs.com/package/botbuilder-azure/)

Reply:
Patched requested changes into [latest commit](https://github.com/Microsoft/botbuilder-js/pull/570/commits/f40fc460e16701924fa315747869dc40af69fb18).